Content.Server/Clothing/Systems/SkaterSystem.cs

```diff
@@ -7,8 +7,8 @@ namespace Content.Server.Clothing;
 
 public sealed class SkaterSystem : EntitySystem
 {
-    [Dependency] private readonly SharedStunSystem _stunSystem = default!;
-    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly SharedStunSystem _stun = default!;
```

Comment:
noob they still not sorted :trollface:

Reply:
then i dont understand what sorted means :)

Reply:
I goes before S